src/components/modules/ApplePayButtonView.res

```diff
@@ -0,0 +1,40 @@
+open ReactNative
+
+type nativeSyntheticEvent<'a> = {nativeEvent: 'a}
+
+type prop = {
+  buttonType?: string, // (buy, setUp, inStore, donate, checkout, book, subscribe)default-> plain
+  buttonStyle?: string, // (white, whiteOutline)default->black
+  cornerRadius?: float,
+  width?: float,
+  height?: float,
+  requestData: string,
+  onPaymentResultCallback: nativeSyntheticEvent<Dict.t<JSON.t>> => unit,
+}
+
+module ApplePayButton = {
+  type props = prop
+  let make = NativeModules.requireNativeComponent("ApplePayView")
+}
+
+@react.component
+let make = (
+  ~buttonType: option<string>=?,
+  ~buttonStyle: option<string>=?,
+  ~cornerRadius: option<float>=?,
+  ~width: option<float>=?,
+  ~height: option<float>=?,
+  ~requestData: string,
+  ~onPaymentResultCallback: nativeSyntheticEvent<Dict.t<JSON.t>> => unit,
+) => {
+  let props: prop = {
+    ?buttonType,
+    ?buttonStyle,
+    ?cornerRadius,
+    ?width,
+    ?height,
+    requestData,
+    onPaymentResultCallback,
+  }
+  <ApplePayButton {...props} />
+}
```

Comment:
Looks like the wrapper component isn't doing anything, so it can be deleted. The content of `ApplePayButton` submodule can be defined at the top level, and that should be sufficient